### PR TITLE
LPD-37690 Fix poshi tests

### PR DESCRIPTION
**Pull request by robertoDiaz**
# What is this trying to solve?

Fixing some test in our CI related with Depots and Staging

# How am I fixing it?

Rendering the right groups in the Asset Publisher configuration to avoid misconfigurations

# How can you verify that it works?

This pr fixes broken tests similar to:

- LocalFile.DepotLocalStagingDM#CanEditDocument

### CODE REVIEW

**Comment by robertoDiaz** [comment]
ci:test:content-management

File: modules/apps/depot/depot-web/src/main/java/com/liferay/depot/web/internal/util/DepotAdminGroupSearchProvider.java [hunk 1]
```diff
@@ -64,6 +64,25 @@ public GroupSearch getGroupSearch(
 		return _getGroupSearch(null, portletRequest, portletURL);
 	}
 
+	private List<Group> _digestGroups(Group group, List<Group> groups) {
```
Comment (adolfopa) [inline modules/apps/depot/depot-web/src/main/java/com/liferay/depot/web/internal/util/DepotAdminGroupSearchProvider.java:67]: Can you rename the method to `_processGroups`? "digest" is normally used in the context of hashing, so it sounds weird here.

**Review by adolfopa — CHANGES_REQUESTED** [review]

File: modules/apps/depot/depot-web/src/main/java/com/liferay/depot/web/internal/util/DepotAdminGroupSearchProvider.java (hunk 1 above)
Comment (robertoDiaz) [reply modules/apps/depot/depot-web/src/main/java/com/liferay/depot/web/internal/util/DepotAdminGroupSearchProvider.java:67]: Sure!

I'll wait for the content-management test to finish and then I'll rename that!

**Comment by robertoDiaz** [comment]
@adolfopa, 

I've updated the method name, could we forward this?

File: modules/apps/depot/depot-web/src/main/java/com/liferay/depot/web/internal/util/DepotAdminGroupSearchProvider.java [hunk 2]
```diff
@@ -160,6 +166,25 @@ private GroupSearch _getGroupSearch(
 		return groupSearch;
 	}
 
+	private List<Group> _processGroups(Group group, List<Group> groups) {
+		if (!group.isStagingGroup()) {
+			return groups;
+		}
+
+		List<Group> digestedGroups = new ArrayList<>();
```
Comment (adolfopa) [inline modules/apps/depot/depot-web/src/main/java/com/liferay/depot/web/internal/util/DepotAdminGroupSearchProvider.java:174]: I think Brian won't like this "digestedGroups" variable name, but I'll leave it to you.

Comment (robertoDiaz) [reply modules/apps/depot/depot-web/src/main/java/com/liferay/depot/web/internal/util/DepotAdminGroupSearchProvider.java:174]: I miss this variable, I'll change it

**Review by adolfopa — APPROVED** [review]

**Comment by adolfopa** [comment]
ci:forward

**Comment by adolfopa** [comment]
ci:test:stable

**Comment by boton** [comment]
ci:test:relevant

**Comment by jkappler** [comment]
ci:forward

**Pull request closed without merging**